cesium/features/lomb_scargle.py

```diff
@@ -19,11 +29,20 @@ def lomb_scargle_model(time, signal, error, sys_err=0.05, nharm=8, nfreq=3, tone
     error : array_like
         Array containing measurement error values.
 
-    nharm : int
-        Number of harmonics to fit for each frequency.
+    sys_err : float, optional
+        Defaults to 0.05
+
+    nharm : int, optional
+        Number of harmonics to fit for each frequency. Defaults to 8.
+
+    nfreq : int, optional
+        Number of frequencies to fit. Defaults to 3.
+
+    tone_control : float, optional
+        Defaults to 5.0
 
-    nfreq : int
-        Number of frequencies to fit.
+    opt_normalize : boolean, optional
+        Defaults to False.
```

Comment:
What does opt_normalize do?

We also need tests for these new flags.

Reply:
The normalization resolves the numerical overflow observed for instance in issue [https://github.com/cesium-ml/cesium/issues/284](#284)


If opt_normalize==False, this refers to the original Lomb-Scargle periodogram function.

Otherwise:
Step 1 - the time-serie (TS) is normalized, and the (mean, std scale) are stored
Step 2 - Lomb-Scargle periodogram & the best fitted model are computed on the normalized (signal, error)
Step 3 - Results are scaled back to the original/raw format using the stored (mean, scale)


**Step  1** 
In function `lomb_scargle_model()`:
```
### Normalization ###
    mmean  = np.nanmean(signal)
    mscale = np.nanstd(signal)
    if opt_normalize:
        signal = (signal-mmean)/mscale
        error  = error/mscale
```

**Step  2**
The lomb-scargle (LS) periodogram is estimated on the normalized time-serie (signal, associated error).

**Step 3**
The best fitted Lomb-Scargle model is rescaled back.
In function `lomb_scargle_model()` :
```
current_fit = rescale_lomb_model(fit, mmean, mscale) if (opt_normalize&(i==0)) else fit 
## function rescale_lomb_model() is used to rescale the best fitted model (computed on normalized TS)
```
The first component (i==0) of `freq_fits` requires to re-inject the scale
```
if (opt_normalize&(i==0)):
            model_dict['freq_fits'][-1]['resid']*=mscale
```